lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -84,6 +84,9 @@ sub test_network_interface {
     my $routed   = $args{routed}   // 0;
     my $target   = $args{target}   // script_output("dig +short openqa.suse.de");
 
+    record_soft_failure('bsc#1177879');
```

Comment:
I would also append bug summary to bug id.

Reply:
sure, will add this as your mentioned.